react-components/src/components/Architecture/pointsOfInterest/useCommentsForPoI.ts

```diff
@@ -0,0 +1,17 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+import { useQuery, type UseQueryResult } from '@tanstack/react-query';
+import { type PointOfInterest, type PointsOfInterestDomainObject } from '../../../architecture';
+import { type CommentProperties } from '../../../architecture/concrete/pointsOfInterest/models';
+import { queryKeys } from '../../../utilities/queryKeys';
+
+export function useCommentsForPoI(
+  domainObject: PointsOfInterestDomainObject<unknown>,
+  poi: PointOfInterest<unknown>
+): UseQueryResult<CommentProperties[]> {
+  return useQuery({
```

Comment:
In other parts of the code (Search UI, for instance) whenever we use the useQuery hook, we are moving the hooks to a subfolder called query and adding a sufix, something like: `useCommentsForPoIQuery`.
Are we gonna follow the same pattern here?

Reply:
Hmm - sure - I can do it - but I don't know if it makes that much sens, there aren't that many hooks in this folder after all. But I think I'll rename it `*Query`